src/renderers/shared/stack/reconciler/__tests__/ReactUpdates-test.js

```diff
@@ -506,6 +508,9 @@ describe('ReactUpdates', () => {
   });
 
   it('should share reconcile transaction across different roots', () => {
+    if (ReactDOMFeatureFlags.useFiber) {
```

Comment:
This was added in ddf53f97f09a652f173bc509b903ead920d92b52 and is just a perf thing. We could probably just delete it.